src/modules/users-profile/index.js

```diff
@@ -9,8 +9,12 @@ export default class UsersProfile extends HasUserModel {
     
     templater.default().template(__dirname+"/../../../templates/user-profile.ejs", "page")
 
-    router.route('GET', '/profile', ::this._profileHandler)
-    router.route('POST', '/profile/save', ::this._saveProfile)
+    let baseUrl = application.config['users'].baseUrl
+
+    if(baseUrl[baseUrl.length-1] != '/') baseUrl += '/'
```

Comment:
This looks like logic that needs to live somewhere that all of these can use. Should the parent module provide a method for accessing baseUrl to address this and all submodules poking into the parent config?